### PR TITLE
Look for and pass proper command line arguments to vswhere

### LINKED ISSUE #2: Takes too long (18 secs)
This action simply finds the path to msbuild using the `vswhere` tool - why on earth it's taking 18 seconds? Is it because of the slow file system on the Github-hosted runner?

### LINKED ISSUE #10: The -find parameter is too new to use safely
I added the `-find` parameter in vswhere 2.7, which wasn't included in Visual Studio until recently. It's not safe to use for hosts with older Visual Studio versions. There is https://github.com/marketplace/actions/setup-vswhere-exe that devs could call first. You might recommend this in the README, but it's probably just safer to stick with the older parameters and look for MSBuild yourself, which shouldn't be too hard since there are nice globbing functions available to node - which were actually my inspiration for adding that functionality in vswhere for non-node callers, like batch scripts that are more common.

### PR DESCRIPTION
**Pull request by heaths**
Fixes #2 and fixes #10

### CODE REVIEW

**Comment by heaths** [comment]
@timheuer take PR #9 then #11 before this. I can rebase anything if there's an issue or, if you prefer, just squash it all and submit a single PR. Originally I was just going to fix one issue and keep the PR focused, then I found another issue to fix, and another... 😊

Also, each of these uses semver since these are all bug fixes and shouldn't affect any public usage. I'm not sure how well that jives with GitHub actions. Seems it's concept of v1, v2, etc., is monotonic and independent of packages' semvers; but, I'm happy to make any adjustments as desired.

**Comment by heaths** [comment]
Ping

File: README.md [hunk 1]
```diff
@@ -3,27 +3,29 @@ You know how handy that 'Visual Studio Developer Command Prompt' is on your loca
...
+You may have a situation where your Actions runner has multiple versions of Visual Studio and you need to find a specific version of the tool.  Simply add the `vs-version` input to specify the range of versions to find.  If looking for a specific version, specify the minimum and maximum versions as shown in the example below, which will look for just 16.4.
 
-```
+```yml
 - name: Add msbuild to PATH
-  uses: microsoft/setup-msbuild@v1.0.0
+  uses: microsoft/setup-msbuild@v1.0.1
     with:
-      vs-version: [16.4,16.5]
+      vs-version: '[16.4,16.5)'
```
Comment (thomthom) [inline README.md:18]: Is there any syntax that allows for open ended versions? Say, if I'm ok with in the 16.x ranges from 16.4 and up?

Comment (thomthom) [reply README.md:18]: Never mind, I looked closer at the vswhere help. `'[16.0,17.0)'` is what I'm looking for.

**Comment by heaths** [comment]
Any change to take a look at this? I even added tests and they all pass.

**Comment by heaths** [comment]
Would you like me to add even more tests to raise confidence in this change? There are significant bugs in how vswhere is used now that this fixes.

**Comment by Timic3** [comment]
Please, merge this, otherwise this Action is absolutely useless. I'm using this fork instead of the official one and it works great.

Thanks for the contribution!

**Comment by Timic3** [comment]
Thanks for the merge and great work. I hope my previous comment did not come off as disrespectful. :)

**Comment by timheuer** [comment]
Maybe a bit :-) -- but it was enough of a nudge to compliment Heath's nudges as well.

**Comment by Timic3** [comment]
I am sorry about that, I do appreciate all the work you put in though, and it's open source after all, so yeah... :) Thanks again and have a nice day!

**Comment by timheuer** [comment]
No worries!

**Pull request merged**